spacy_llm/tasks/summarization/registry.py

```diff
@@ -22,7 +22,7 @@ def make_summarization_task(
     parse_responses (Optional[TaskResponseParser[SummarizationTask]]): Callable for parsing LLM responses for
         this task.
     prompt_example_type (Optional[Type[FewshotExample]]): Type to use for fewshot examples.
-    examples (Optional[Callable[[], Iterable[Any]]]): Optional callable that reads a file containing task examples for
+    examples (ExamplesConfigType): Optional callable that reads a file containing task examples for
```

Comment:
Do we need to include these changes to unrelated tasks in this PR? It makes reviewing a bit more complicated.

Reply:
We don't need to. Let me know if I should revert.

Reply:
Done